extensions/sequence.md

```diff
@@ -20,11 +20,11 @@ event producer to understand how to interpret the value of the attribute.
 * Constraints
   * REQUIRED
   * MUST be a non-empty lexicographically-orderable string
-  * RECOMMENDED as contiguous
+  * RECOMMENDED as montomically contiguous
```

Comment:
There is a typo, maybe `monotonically increasing and contiguous` reads better?

My math is quite rusty, but I'm not sure if a char-based sequence can legitimately be called `monotonically increasing` (e.g. `a, b, ...z, aa, ab, ...`). But, as it is only a recommendation, I think it's fine to get the point across.

Reply:
done!